src/relay/backend/contrib/arm_compute_lib/codegen.cc

```diff
@@ -126,7 +127,7 @@ class ACLJSONSerializer : public backend::contrib::JSONSerializer {
       nodes.activation = current_call;
       current_call = current_call->args[0].as<CallNode>();
     }
-    if (backend::IsOp(current_call, "nn.bias_add")) {
+    if (backend::IsOp(current_call, "add")) {
```

Comment:
I remember needing to change this but I didn't get to the bottom of why. Is there an explanation?

Reply:
It will be "lowered" (not sure I am using correct terminology) to add on time when the runtime is invoked